vote/vote.py

```diff
@@ -58,6 +60,31 @@ def get_votes(self):
 
         return votes
 
+    def get_least_voted(self, items, num=2):
+        """ Select (num) number of the least-voted instances
+
+        >>> v = MemeVote(test_mode=True)
+        >>> v.vote('alpha', 'beta', 'alpha')
+        >>> v.vote('beta', 'gamma', 'gamma')
+        >>> v.vote('gamma', 'delta', 'gamma')
+        >>> v.vote('alpha', 'gamma', 'alpha')
+        >>> v.get_least_voted(['alpha', 'beta', 'gamma', 'delta', 'omega'])
+        ['omega', 'delta']
+        """
+        searches = [vote['meme1'] + '|' + vote['meme2'] for vote in self.get_votes()]
+        tally = {}
+        for item in items:
+            tally[item] = len([x for x in searches if item in x])
```

Comment:
This is a tally of how many times an instance has been voted on. I don't know what 'meme to comparison count' means. We'll have to agree to disagree about clarity. 

String collision was a consideration. The method that guaranteed even more unique strings was rejected. Python's tempfiles method gives us input strings like 'output/<url-name>.<six-character-string>.jpg' which has MAX_GENERATORS * 38^6 permutations *at worst*. Substrings are guarded by an invalid path character. I am as concerned with this as UUID collision or cosmic rays flipping a bit.